shambomon/assets/js/components/attack.jsx

```diff
@@ -4,6 +4,18 @@ import React from 'react';
 export default function Attack(props) {
   let players = props.state.players;
   let disabled = players[props.state.turn].id != props.id;
+  let specialDisabled = players[props.state.turn].specialUsed || disabled
+  let specialSet = false;
+
+  function attack(option) {
+    console.log(specialSet);
```

Comment:
Need to remove this print statement before pushing to master

Reply:
Yeah, I removed it in another branch that I'll merge into this.